great_expectations/rule_based_profiler/data_assistant/data_assistant.py

```diff
@@ -306,57 +303,58 @@ def get_expectation_suite(
     ) -> ExpectationSuite:
         """
         Args:
-            expectation_suite: An existing "ExpectationSuite" to update.
-            expectation_suite_name: A name for returned "ExpectationSuite".
+            expectation_suite: An existing "ExpectationSuite" to update
+            expectation_suite_name: A name for returned "ExpectationSuite"
             include_citation: Whether or not to include the Profiler config in the metadata for "ExpectationSuite" produced by "RuleBasedProfiler"
 
         Returns:
-            "ExpectationSuite" using "ExpectationConfiguration" objects, computed by "RuleBasedProfiler" state.
+            "ExpectationSuite" using "ExpectationConfiguration" objects, computed by "RuleBasedProfiler" state
```

Comment:
@NathanFarmer are periods at the end of these docstring components out of compliance with our style guide?  Thanks (just wondering; no big deal).

Reply:
@alexsherstinsky I just prefer it and did the rest of them without the periods. I will look into it.